### DISCUSSION TITLE
Help with error trying to use Cake in GitHub actions

### DISCUSSION THREAD

**Discussion by musictopia2** [cake-build/cake#3185] [Q&A] [answered]
I created the most simple .net project.  The build processes works perfectly when using visual studio code and using the terminal to type in the command dotnet cake.  However, when running on github, does not work at all.
My repository is here
https://github.com/musictopia2/Solution1 

You can see if you go to build actions, you can see that even though I have all the code necessary, it still will not work with github.  I hope there is a way to make it work somehow with github.

**Comment by patriksvensson** [comment, marked as answer]
[This line](https://github.com/musictopia2/Solution1/blob/master/.github/workflows/dotnet.yml#L21) should be `dotnet tool restore`.

You're getting an error because you're invoking `dotnet restore tools` which tries to do something completely different than restoring tools

**Reply by musictopia2** [reply]
That worked perfectly.  Thanks.  The previous line was based on a comment made who thought that was the code for the restoring tools.
